src/layouts/BaseLayout/hooks/useUserAccessGate.js

```diff
@@ -1,34 +1,10 @@
-import { useHistory, useParams, useRouteMatch } from 'react-router-dom'
+import { useParams } from 'react-router-dom'
 
 import config from 'config'
 
-import { useUpdateDefaultOrganization } from 'services/defaultOrganization'
-import { useLocationParams } from 'services/navigation'
 import { useUser } from 'services/user'
 import { useFlags } from 'shared/featureFlags'
 
-const SetUpActions = Object.freeze({
-  INSTALL: 'install',
-  REQUEST: 'request',
-})
-
-function useOnboardingRedirect({ username }) {
-  const history = useHistory()
-  const matchProvider = useRouteMatch('/gh')
-  const { params } = useLocationParams()
-  const { mutate: updateDefaultOrg } = useUpdateDefaultOrganization()
-
-  if (!username) return
-
-  const { setup_action: setupAction } = params
-
-  if (setupAction === SetUpActions.REQUEST && matchProvider.isExact) {
-    updateDefaultOrg({ username })
-
-    return history.push(`/gh/${username}?setup_action=${SetUpActions.REQUEST}`)
-  }
-}
-
 // eslint-disable-next-line complexity, max-statements
```

Comment:
Does this need to be here now that we got rid of the onboarding hook?

Reply:
yes unfortunately